wiki/Ranking_criteria/osu!mania/en.md

```diff
@@ -58,7 +58,7 @@ Overall rules and guidelines apply to every kind of osu!mania difficulty. Rhythm
 - **Two notes cannot be placed in a single column at the same timestamp.**
 - **No column can be left empty.** If you want to use fewer keys, change the number of keys in the Song Setup.
 - **There must not be more than 6 notes pressed at any given moment in Insane or lower difficulties.** Using more than 6 notes must also follow a reasonable spread to the next lower difficulty. This rule does not apply to ends of long notes, as they are released, not pressed.
-- **Beatmaps can only use 4 to 10 keys.** Anything else is not supported for the Ranked section.
+- **Beatmaps can only use 4–10, 12, 14, 16, or 18 keys.** Anything else is not supported for the Ranked section.
 - **If multiple key modes are present in a single beatmap, the key mode must be denoted in all difficulty names. Otherwise, the key mode must not be denoted.**
 - **Beatmaps using the special N+1 style of mapping must be denoted as such under the `Advanced` tab of song setup in the editor and "N+1" must be added to the tags. For beatmaps containing multiple key modes, difficulties using this style must be denoted using N+1 notation (e.g. using 7K+1 instead of 8K on difficulty names).**
```

Comment:
I don't feel strongly about this either way, but you might want to reword this to be more generic than "N+1" now that you could rank DP special styles

Reply:
i'm not sure what that is or if/where it's mentioned in the rc

can you be a bit more concrete? are you referring to another proposal?

Reply:
N+1 purpose was to differentiate two distinct map styles sharing same key count in 4-10K. Mostly it refers to differentiating 7K+1 and 8K, due to both are quite popular modes to map on these keycount, but also covers anything else like 5K+1 and 6K and novel styles like 4K+1. https://osu.ppy.sh/community/forums/topics/1694238

Due to how broad the playstyles offered by 12K+, the ruling about key count in diff names can be enforced for 12K+, which also extends to mentioning the playstyle it offered. N+1 should remain there if there is only one special key (on left column) present in the map. 4-10K descriptor should remain unmentioned unless multiple key modes present.

Reply:
as an update people seem to be working on an updated wording to this rule and a new [supplementary article](https://discord.com/channels/188630481301012481/218677502141399041/1161131392701309038) that this rule can link to